src/sonic-yang-models/yang-models/sonic-mgmt_interface.yang

```diff
@@ -0,0 +1,63 @@
+module sonic-mgmt-interface {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-mgmt-interface";
+    prefix mgmtintf;
+
+    import sonic-mgmt-port {
+        prefix mgmtprt;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description
+        "SONiC MANAGEMENT INTERFACE";
+
+    revision 2019-11-13 {
+        description "First revision";
+    }
+
+    container sonic-mgmt-interface {
+
+        container MGMT_INTERFACE {
+
+            description "MANAGEMENT INTERFACE part of config_db.json";
+
+            list MGMT_INTERFACE_LIST {
+
+                key "name ip_prefix";
+
+                leaf name{
+                    type leafref {
+                        path /mgmtprt:sonic-mgmt-port/mgmtprt:MGMT_PORT/mgmtprt:MGMT_PORT_LIST/mgmtprt:name;
+                    }
+                }
+
+                leaf ip_prefix {
+                    must "(contains(current(), ':') and contains(../gwaddr, ':')) or (contains(current(), '.') and contains(../gwaddr, '.'))";
+                    type stypes:sonic-ip-prefix;
+                }
+
+                leaf gwaddr {
+                    must "(contains(current(), ':') and contains(../ip_prefix, ':')) or (contains(current(), '.') and contains(../ip_prefix, '.'))"; 
+                    type inet:ip-address;
+                }
+
+                leaf-list forced_mgmt_routes {
+                    type stypes:sonic-ip-prefix;
```

Comment:
Please add a description for this leaf-list.

Reply:
Done.

Reply:
Is this covered? Just the IP?

```
            "forced_mgmt_routes": [
                "10.3.145.14",
```